test/serviceprofiles/serviceprofiles_test.go

```diff
@@ -1,16 +1,26 @@
 package serviceprofiles
 
 import (
+	"errors"
 	"fmt"
 	"os"
 	"strings"
 	"testing"
+	"time"
 
+	sp "github.com/linkerd/linkerd2/controller/gen/apis/serviceprofile/v1alpha1"
 	"github.com/linkerd/linkerd2/testutil"
+	"sigs.k8s.io/yaml"
 )
 
 var TestHelper *testutil.TestHelper
 
+type rowStat struct {
```

Comment:
TIOLI, you could make `jsonRouteStats` (from the `cmd` package) public and re-use that.